docs/allocations.md

```diff
@@ -0,0 +1,26 @@
+# Allocations
+
+Each year, around the same time as [Rollover](./rollover.md), providers request
+places for courses that have numbers of applicants limited as they are generally
+over-supplied. This process is called 'Allocations'.
+
+This document lists the changes needed to be made to the Publish codebase and
+the timings for these changes.
+
+## Before Allocations
+
+- Update any hardcoded dates relating to Allocation years and timelines
```

Comment:
I think we need to make some references to academic year

Reply:
done!